git/patch.go

```diff
@@ -143,14 +143,6 @@ func parsePatchHeader(b []byte) (Patch, error) {
 	if err != nil {
 		return Patch{}, err
 	}
-	addrs, err := m.Header.AddressList("From")
-	if err != nil {
-		return Patch{}, err
-	}
-	if len(addrs) != 1 {
-		return Patch{}, errMalformedPatch
-	}
-	p.Author = addrs[0].String()
```

Comment:
Can we use `m.Header.Get("From")` instead? Is there some case (multiple senders?) that `git show` will handle correctly but this won't?

Reply:
That's a good idea. I'll do that. My understanding is that [co-authorship is handled by having `Co-authored-by:` in the commit message](https://stackoverflow.com/questions/7442112/how-to-attribute-a-single-commit-to-multiple-developers).

Reply:
Sounds good. It might be a good idea to add a regression case for this too (or just replace the `marius@` test with `dependabot[bot]`).

Reply:
I added a test.